sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -94,4 +112,24 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         );
     }
   }
+
+  /**
+   * Updates the infected file status and renames file to describe an error with the file.
+   * @param studentFile student infected file to be updated.
+   * @returns the update result.
+   */
+  private async updateInfectedFile(
+    studentFile: StudentFile,
+  ): Promise<UpdateResult> {
+    const fileName = path.parse(studentFile.fileName).name;
+    const fileExtension = path.parse(studentFile.fileName).ext;
+
+    return this.repo.update(
+      { id: studentFile.id },
+      {
+        fileName: `${fileName}${INFECTED_FILENAME_SUFFIX}${fileExtension}`,
```

Comment:
In the shared images in the PR description, the files are named with the extension twice, for instance: "eicar.txt-OriginalFileError.txt". Would "eicar-OriginalFileError.txt" be the expected?

Reply:
Sorry, please, disregard those examples in the PR description. That was about some of my previous tests. The name is ok now. I'm going to fix the image.